module/python/bench_ebpf_space.py

```diff
@@ -0,0 +1,144 @@
+import argparse
+import os
+from ctypes import c_int
+
+from bcc import BPF
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-n', action='store', dest='number', help="Number of iterations", default=1000)
```

Comment:
I think this is missing a type=int; crashes without one

Reply:
Good catch!